### PR TITLE
feat(TBD-5000): Add Spark 2.1 in local mode

### PR DESCRIPTION
**Pull request by zafkirTalend**
We build the Spark Cassandra Connector against Scala 2.11 (spark-cassandra-connector-assembly-2.0.2-patched-20170519.jar is generated).
Please see https://github.com/Talend/spark-cassandra-connector/blob/master/doc/12_building_and_artifacts.md

### CODE REVIEW

File: main/plugins/org.talend.libraries.apache.cassandra/plugin.xml [hunk 1]
```diff
@@ -47,6 +47,12 @@
             mvn_uri="mvn:org.talend.libraries/spark-cassandra-connector-assembly-2.0.0-M3-patched-20161017/6.3.0"
             name="spark-cassandra-connector-assembly-2.0.0-M3-patched-20161017.jar">
       </libraryNeeded>
+      <libraryNeeded
+            context="plugin:org.talend.libraries.apache.cassandra"
+            id="spark-cassandra-connector-21"
+            mvn_uri="mvn:org.talend.libraries/spark-cassandra-connector-assembly-2.0.2-patched-20170519/6.3.0"
```
Comment (pbailly) [inline main/plugins/org.talend.libraries.apache.cassandra/plugin.xml]: Could you rename the jar into spark-cassandra-connector-assembly_2.11-2.0.2.jar?

Comment (pbailly) [inline main/plugins/org.talend.libraries.apache.cassandra/plugin.xml]: Change the 6.3.0 into 6.0.0

Comment (zafkirTalend) [inline main/plugins/org.talend.libraries.apache.cassandra/plugin.xml]: Done

Comment (zafkirTalend) [inline main/plugins/org.talend.libraries.apache.cassandra/plugin.xml]: Done

Comment (bchen-talend) [inline main/plugins/org.talend.libraries.apache.cassandra/plugin.xml]: we'd better let others know this jar has been patched by us somewhere, comment or jar's name

Comment (zafkirTalend) [inline main/plugins/org.talend.libraries.apache.cassandra/plugin.xml]: @bchen-talend done

**Pull request merged**